src/main/java/io/kokuwa/maven/helm/InstallMojo.java

```diff
@@ -31,6 +31,22 @@ public class InstallMojo extends AbstractHelmWithValueOverrideMojo {
 	@Parameter(property = "action", defaultValue = "install")
 	private String action;
 
+	/**
+	 * Set this to <code>true</code> to delete the installation on failure.
+	 *
+	 * @since 6.10.0
+	 */
+	@Parameter(property = "helm.install.atomic")
+	private boolean installAtomic;
+
+	/**
+	 * Time to wait for any individual Kubernetes operation.
+	 *
+	 * @since 6.10.0
+	 */
+	@Parameter(property = "helm.install.timeout")
+	private String installTimeout;
```

Comment:
Should we use `Integer` here and and pass it as seconds to helm. This can prevent unparsable input.

@fabian-schlegel  @nickkalgin What's your opinion?

Reply:
I guess this is about "5m" vs. "300". In principal, I'm a fan of everything that doesn't need parsing. On huge duration the "5m" notation is more readable, but I doubt that anyone will put more than 60 seconds here. So, I'd vote for the integer value.

Reply:
I agree. It's a good idea to use seconds here. Helm requires a time unit declaration, but it's pretty easy to solve.